### PR TITLE
Now getting status of the job from Nomad

### PR DESCRIPTION
**Pull request by gogeterobert**
Signed-off-by: Robert Gogete <gogeterobert@yahoo.com>

### CODE REVIEW

File: src/Infrastructure/Services/NomadJobService.cs [hunk 1]
```diff
@@ -180,13 +181,27 @@ private Fermyon.Nomad.Model.Task GenerateJobTask(NomadJob nomadJob)
...
+    public string GetJobStatus(string jobName)
     {
-        return _jobsClient.GetJobs().Any(job => job.Name == jobName);
+        try
+        {
+            var job = _jobsClient.GetJob(jobName.ToLower());
+
+            if (job == null)
+            {
+                throw new NotFoundException($"Job {jobName} not found");
```
Comment (DanBadea) [inline src/Infrastructure/Services/NomadJobService.cs]: return JobStatus.Dead

File: src/Infrastructure/Services/NomadJobService.cs [hunk 2]
```diff
@@ -180,13 +181,27 @@ private Fermyon.Nomad.Model.Task GenerateJobTask(NomadJob nomadJob)
...
             {
                 { "command", nomadJob.spinBinaryPath },
-                { "args", new List<string> { "up", "--listen", "${NOMAD_IP_http}:${NOMAD_PORT_http}", "--follow-all", "--bindle", nomadJob.BindleId } }
+                { "args", new List<string> { "up", "--listen", "[${NOMAD_IP_http}]:${NOMAD_PORT_http}", "--follow-all", "--bindle", nomadJob.BindleId } }
             }
         };
     }
 
-    private bool DoesJobExist(string jobName)
+    public string GetJobStatus(string jobName)
```
Comment (DanBadea) [inline src/Infrastructure/Services/NomadJobService.cs]: convert to enum the return type

File: src/Infrastructure/Services/NomadJobService.cs [hunk 3]
```diff
@@ -180,13 +181,27 @@ private Fermyon.Nomad.Model.Task GenerateJobTask(NomadJob nomadJob)
             Config = new Dictionary<string, object>
             {
                 { "command", nomadJob.spinBinaryPath },
-                { "args", new List<string> { "up", "--listen", "${NOMAD_IP_http}:${NOMAD_PORT_http}", "--follow-all", "--bindle", nomadJob.BindleId } }
+                { "args", new List<string> { "up", "--listen", "[${NOMAD_IP_http}]:${NOMAD_PORT_http}", "--follow-all", "--bindle", nomadJob.BindleId } }
```
Comment (DanBadea) [inline src/Infrastructure/Services/NomadJobService.cs]: This should be reverted!

File: src/Web/ClientApp/src/app/components/application/list/list.component.html [hunk 4]
```diff
@@ -41,7 +41,7 @@ <h1 class="title">Applications</h1>
 								<span *ngIf="channel.activeRevisionNumber"><strong>v{{channel.activeRevisionNumber}}</strong></span>
 								<span class="icon">
 									<!-- TODO: adapt color to channel status and add tooltip -->
-									<fa-icon [icon]="faCircle" [ngStyle]="{'color': 'green'}"></fa-icon>
+									<fa-icon [icon]="faCircle" [ngStyle]="{'color': channel.status === jobStatus.Running ? 'green' : 'red'}"></fa-icon>
```
Comment (StefanNedelcu) [inline src/Web/ClientApp/src/app/components/application/list/list.component.html]: Maybe remove the above comment and add a tooltip so the user would know what each color means?

**Pull request merged**